deployments/platform-dev/docker-compose.yml

```
@@ -82,8 +82,25 @@ services:
       - MYSQL_PASSWORD
       - MYSQL_USER
       - MYSQL_DATABASE=pathwar
+    expose:
+      - 3306
+    networks:
+      - internal
+
+  forestadmin:
+    build:
+      context: ../../tool/forestadmin
+      dockerfile: Dockerfile
+    environment:
+      - APPLICATION_PORT=3310
+      - DATABASE_URL=mysql://root:uns3cur3@db:3306/pathwar
```

```suggestion
      - DATABASE_URL=mysql://$MYSQL_USER:$MYSQL_PASSWORD@db:3306/pathwar
```

Since you are updating the deployment's docket-compose file (you should follow all its conventions, here -> get most of the advanced configuration from a .env file and reuse the existing variables)

---

ACK